### ISSUE TITLE
Controlling External Climate Entities 

### ISSUE THREAD

**Issue by tompawompa** [Blackymas/NSPanel_HA_Blueprint#1382]
I currently have a esphome node controlling my HVAC system. It is recognized in home assistant as a climate entity. All of the code is contained on that esp32, using the thermostat platform.

When I select this node as my climate entity in the blueprint, I get a slider that is blank and cannot adjust the setpoint. I also cannot access the presets I have configured. The HVAC mode icons and the corresponding temp sensor are displayed correctly. 
![20231203_205838](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/95145927/592e42f6-37cd-481d-97c7-7db655760fed)

**Comment by edwardtfn** [comment]
Climate controls with 2 set points (max & min) temperatures are not yet supported. Is this your case?

**Comment by tompawompa** [comment]
Yes, it is. As I don't utilize the auto mode too often, I'll reconfigure that node with a single setpoint and report back.

**Comment by edwardtfn** [comment]
This is similar to #909 and #1106.

**Comment by tompawompa** [comment]
After playing around with the code, I gave up. I wasn't able to make a dual mode thermostat, that utilized one setpoint for both modes in esphome.

I do have generic thermostat configured for an aquarium heater, and it appeared to render correctly.

**Comment by edwardtfn** [comment]
I'm planning to add this functionality on v4.2... No promises... 😉

**Comment by tompawompa** [comment]
I hope! Having an upstairs and downstairs thermostat was my main motivator for going with this project 😬

**Comment by edwardtfn** [comment]
Duplicated: #909

**Comment by tompawompa** [comment]
I thought I'd share my work around in the mean time. I've created two template sensors for my high and low set points to display on the climate page. 

I then created two scripts, also using templates, for moving my target low up and down, and used the custom buttons to run. Since their only two buttons I couldn't do the same for target high, but it's winter anyway.
![20231223_093712](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/95145927/f71ac5c9-d3b6-4347-9ede-27c66aa18e03)
